hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/helpers/BlockUtils.java

```diff
@@ -56,24 +56,21 @@ private BlockUtils() {
    * the caller must close it after using it.
    * If another thread attempts to open the same container when it is already
    * opened by this thread, the other thread will get a RocksDB exception.
-   * @param containerID The containerID
    * @param containerDBPath The absolute path to the container database folder
    * @param schemaVersion The Container Schema version
    * @param conf Configuration
    * @return Handler to the given container.
    * @throws IOException
    */
-  public static DatanodeStore getUncachedDatanodeStore(long containerID,
+  public static DatanodeStore getUncachedDatanodeStore(
```

Comment:
Lets modify the comments as well here, as now this may return the DB for the disk. Please modify the comment at line 55 here.